agent/agent.go

```diff
@@ -298,6 +298,44 @@ func (a *Agent) withSession(ctx context.Context, fn func(session *session) error
 	}
 }
 
+// AttachNetwork sends a container attach network request towards a
```

Comment:
Wouldn't this be done as part of the backend? The agent shouldn't really provide this API.